### ISSUE TITLE
Crypto js AES support

### ISSUE THREAD

**Issue by ppbntl19** [kjur/jsrsasign#232]
How can we use crypto AES without including any cryptoJS files.

` CryptoJS.AES.decrypt/encrypt` function  available  globally  but throwing error when try to use encrypt and decrypt.

![image](https://cloud.githubusercontent.com/assets/11989749/21984281/3e6ca796-dc1b-11e6-978f-a23cce100b0c.png)
![image](https://cloud.githubusercontent.com/assets/11989749/21984338/78de5b90-dc1b-11e6-8c60-dbd4409f0777.png)

**Comment by kjur** [comment]
Thank you for your report. I'll check it later. Did you do it on your web browser?

**Comment by ppbntl19** [comment]
Thanks
Yes, I tried it on browser console and  in JS code. But getting same exception.

**Comment by kjur** [comment]
I've investigated it. Please use encrypt and decrypt method of CryptoJS.AES class. I'll show you AES decrypt example here:

```JavaScript```
// parameters
var keyWS = CryptoJS.enc.Hex.parse('4bb9...');
var ivWS = CryptoJS.enc.Hex.parse('79c2...');
var encWS = CryptoJS.enc.Hex.parse('c70b...');

// decrypt
var decWS = CryptoJS.AES.decrypt({ciphertext: encWS}, keyWS, {iv: ivWS});
```

**Comment by getify** [comment]
This is still a bug and should be re-opened. The OP said that they were already using the encrypt/decrypt on the CryptoJS.AES class. The problem appears to be that the copy of CryptoJS that `jsrsasign` ships with is not fully functional on its own.

I just got the latest `jsrsasign`, and I'm having the same exception thrown as the OP. I have some code like this:

```js
var x = CryptoJS.AES.encrypt("Hello World!","this is my secret password");

CryptoJS.AES.decrypt(x.toString(),"this is my secret password").toString(CryptoJS.enc.Utf8);
// "Hello World!"
```

That code works fine if I manually include the full CryptoJS library in the page. But if I only include `jsrsasign` in the page, then `CryptoJS.AES.encrypt(..)` throws an error:

![screen shot 2018-02-14 at 9 59 18 am](https://user-images.githubusercontent.com/150330/36214003-d728c1c6-116d-11e8-9508-ea0f479d4152.png)

I've verified that I first load `jsrsasign` into the page, and then load CryptoJS on top of it, everything works fine. So it really must be that the CryptoJS that is shipped with this library is stripped or not fully functional in some way.
